internal/ingress/controller/template/template.go

```diff
@@ -1377,3 +1378,29 @@ modsecurity_rules_file /etc/nginx/modsecurity/modsecurity.conf;
 
 	return buffer.String()
 }
+
+func buildMirrorLocations(locs []*ingress.Location) string {
+	var buffer bytes.Buffer
+
+	mapped := sets.String{}
+
+	for _, loc := range locs {
+		if loc.Mirror.URI == "" || loc.Mirror.BackendURL == "" {
+			continue
+		}
+
+		if mapped.Has(loc.Mirror.URI) {
+			continue
+		}
+
+		mapped.Insert(loc.Mirror.URI)
+		buffer.WriteString(fmt.Sprintf(`location = %v {
+internal;
+proxy_pass %v;
+}
+
+`, loc.Mirror.URI, loc.Mirror.BackendURL))
```

Comment:
Why are you not considering `mirror_request_body` here?

Reply:
The mirror configuration is inside the location block